recipes/LJSpeech/S2ST/extract_code.py

```diff
@@ -0,0 +1,161 @@
+"""
+Apply K-means clustering over acoustic features.
```

Comment:
Add a bit more context with respect to the recipe. What needs to be done before, and w<hat can be done after.

Reply:
Done